design/style_a_map.md

```diff
@@ -8,39 +8,39 @@ In this lab you will use the styling capabilities of the viewer to make your map
 
 1. Go to [arcgis.com](http://www.arcgis.com) and login. Check `Keep me signed in`.
 
-2. Click on [this link](http://www.arcgis.com/home/webmap/viewer.html?webmap=df0da19116934be89ac5f7379172c24e) to load the unstyled map.
+2. Click on [this link](http://www.arcgis.com/home/webmap/viewer.html?webmap=df0da19116934be89ac5f7379172c24e) to load the unstyled map or use your own unstyled map.
 
 3. Click `Save As` to save your own version of the map. Optionally `Share` > `Everyone`.
 
 4. Click on `Rail Stops` > `Change Style` and set the following:
 	
-	1. Choose an attribute to show: Type
+	1. 1 - Choose an attribute to show: `Type`
  	
```

Comment:
I think we decided to only use ```xxx``` for clickable items. In this case the title label is exactly this: 1. Choose an attribute to show